deps/rabbitmq_management_agent/src/rabbit_mgmt_metrics_gc.erl

```diff
@@ -82,13 +85,13 @@ remove_connection(Id, BIntervals) ->
     delete_samples(connection_stats_coarse_conn_stats, Id, BIntervals),
     ok.
 
-remove_channel(Id, BIntervals) ->
+remove_channel(Id, CMCount, BIntervals) ->
```

Comment:
Any reason not make it a few characters longer and say `ConsumerCount`?

Reply:
No valid reason! I guess my C days still are influencing my var names.